### PR TITLE
fix `t.typeText() not working with React's new version (15.6.1)` (close #1558)

### LINKED ISSUE #1558: t.typeText() not working with React's new version (15.6.1)
### Are you requesting a feature or reporting a bug?

It can be a bug or a feature request, depends on the way you look at it :stuck_out_tongue: 

### What is the current behavior?

For some reason ```t.typeText()``` is not triggering the ```onChange``` handler within a text input element.

### What is the expected behavior?

```t.typeText()``` should trigger ```onChange``` event when typing each character so we can react on it.

### How would you reproduce the current behavior (if this is a bug)?

1. Create an application with React latest version (15.6.1)
2. Create a component with a text input element and setup any handler for the ```onChange``` prop
3. Add a ```console.log()``` in the handler
4. Create a test that types into that text input
5. Run the tests and assert that no logs were emitted

#### Provide the test code and the tested page URL (if applicable)

Test code

Component:
```js
class LoginForm extends PureComponent {
  constructor(props) {
    super(props);

    this.state = {
      email: ''
    };

    this.handleEmailChange = this.handleEmailChange.bind(this);
  }

  handleEmailChange({ target }) {
    console.log(target.value);
    
    this.setState({ email: target.value });
  }

  render() {
    return (
      <input
        id="loginField"
        type="text"
        value={this.state.value}
        onChange={this.handleEmailChange}
      />
    );
  }
}
```

Test
```js
test('Should be able to type into login input', async (t) => {
  await t
    // Will show as if it's working, but nothing will be logged in the console
    .typeText(loginField, 'This will not be logged')
    // Although this will pass
    .expect(loginField.value, 'This will not be logged');

    // So you can can open console and see output
  await t.wait(1000000);
});
```

### Specify your

* operating system: Ubuntu 16.04 LTS
* testcafe version: 0.16.1
* node.js version: 6.11.0

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @AlexanderMoskovkin @helen-dikareva

### CODE REVIEW

File: test/client/fixtures/automation/press-test.js [hunk 1]
```diff
@@ -245,4 +248,41 @@ $(document).ready(function () {
...
+            var testNativeValueSetter = function (element, callback) {
+                var valueGetter = Object.getOwnPropertyDescriptor(element.constructor.prototype, 'value').get;
+                var press = new PressAutomation(parseKeySequence('backspace').combinations, {});
+
+                element.value = '1';
+                element.focus();
+
+                Object.defineProperty(element, 'value', {
+                    get: function () {
+                        return valueGetter.call(element, '1');
```
Comment (helen-dikareva) [inline test/client/fixtures/automation/press-test.js]: the `'1'` argument is redundant

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
rebase required

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
An unstable test is failed (it doesn't relate to the PR), so I merge it.

**Pull request merged**